js/src/mesh.ts

```diff
@@ -90,11 +132,21 @@ class MeshView extends widgets.WidgetView {
 
         this.create_mesh();
         this.add_to_scene();
+        
         this.model.on("change:color change:sequence_index change:x change:y change:z change:v change:u change:triangles change:lines",
             this.on_change, this);
         this.model.on("change:geo change:connected", this.update_, this);
         this.model.on("change:texture", this._load_textures, this);
         this.model.on("change:visible", this.update_visibility, this);
+        this.model.on("change:lighting_model change:opacity change:specular_color change:shininess change:emissive_color change:emissive_intensity change:roughness change:metalness change:cast_shadow change:receive_shadow", 
+        this.update_visibility, this);
+    }
+
+    public force_lighting_model() {
+        if(this.lighting_model === this.LIGHTING_MODELS.DEFAULT){
+            this.model.set("lighting_model", this.LIGHTING_MODELS.PHYSICAL);
+            this.update_visibility();
```

Comment:
```suggestion
            this. _update_materials();
```
I guess that should be enough right?

Reply:
fixing